src/widgets/expo-detail/ui/ExpoActionPanel/index.tsx

```diff
@@ -45,9 +47,11 @@ const ExpoActionPanel: React.FC<ExpoActionPanelProps> = ({
 
   return (
     <div>
+      <ClientInitializer />
+
       <div className="h-fit max-w-[210px] space-y-[26px] rounded-sm border-1 border-solid border-gray-200 p-[18px] mobile:max-w-full mobile:border-none mobile:px-[16px]">
         <p className="text-caption1 text-black mobile:hidden">
-          2024 AI광주미래교육박람회
+          2024 AI광주미래교육박람회 {role}
```

Comment:
```suggestion
          2024 AI광주미래교육박람회
```
role 테스트 하고 실수로 안지웠습니다 ㅠ